Yosemite/Yosemite/PointOfSale/Items/POSSimpleProduct.swift

```diff
@@ -19,6 +19,9 @@ public struct POSSimpleProduct: POSOrderableItem, OrderSyncProductTypeProtocol {
     public let manageStock: Bool
     public let stockQuantity: Decimal?
     public let stockStatusKey: String
+    public var productStockStatus: ProductStockStatus {
+        return ProductStockStatus(rawValue: stockStatusKey)
+    }
```

Comment:
nit: this probably doesn't need to be a public extension, as the current usage is just within `StockFormatter`.

Reply:
Since this is in Yosemite and the usage of StockFormatter in WooCommerce it doesn't compile if we make it internal.